src/redux/rockets/rocketThunk.js

```diff
@@ -0,0 +1,11 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+const fetchRocket = createAsyncThunk('rocket/fetchRockets', async () => {
+  const response = await fetch('https://api.spacexdata.com/v3/rockets', {
+    method: 'GET',
+  });
+
+  return response.json();
```

Comment:
- [ ] Please do the same thing for the rockets' data as discussed earlier. ✍️

**Necessary data for the Rockets component**
![Screenshot from 2023-04-13 11-54-10](https://user-images.githubusercontent.com/17243325/231713472-aa94504d-f3df-4464-9aa2-bf2f79900307.png)